src/main/resources/application.properties

```diff
@@ -14,6 +14,9 @@ spring.datasource.password=
 
 ## Spring Liquibase
 spring.liquibase.change-log=classpath:liquibase/db.changelog-master.xml
+spring.liquibase.url=jdbc:mysql://localhost:3306/evo_exchange?serverTimezone=UTC&createDatabaseIfNotExist=true&useSSL=false
```

Comment:
You should remove all liquibase proprerties except ``spring.liquibase.change-log``. Spring Boot invokes them by default under the hood.

Reply:
Ok